themes/default/content/docs/pulumi-cloud/oidc/_index.md

```diff
@@ -51,26 +66,37 @@ Valid custom claims for this service are listed in the table below:
 
 ### Pulumi ESC
 
-The format of the subject claim for this service is:
+The default format of the subject claim for this service is:
 
 `pulumi:environments:org:<organization name>:env:<environment name>`
 
+However, we recommend using `subjectAttributes` property to customize the OIDC subject claims if you plan to use subject claims in your cloud provider trust policy. The default prefix when using `subjectAttributes` will be
+
+`pulumi:environments:pulumi.organization.login:{ORGANIZATION_NAME}`
+
+Additional options for customization include:
+
+* `rootEnvironment.name`: the name of the environment that is opened first. This root environment in turn opens other imported environments
```

Comment:
Might want to make clear that for IaC situations this will be the rootest that is not the IaC env.

Reply:
How would we phrase it?